src/types/api/package.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
I don't know if this will necessarily cause issues, but putting this inside the desktop app's source tree code would cause issues. We ran into some problems with the web app repo when we moved mattermost-redux into it, but that might have been because we were trying to use workspaces and some other weirdness

Reply:
I did originally have it outside of the `src` folder, but it was a pain when developing so I moved it in.

If it's going to cause problems then I'll move it back out and try the `file:` method again.

Though for my sake, getting the types published sooner rather than later would be helpful ;)

Reply:
I'm more thinking about making development in the desktop app easier rather than how the web app would use it.

I think the "correct" way would be to have the desktop repo be a monorepo with multiple workspaces, but I don't know if there's any real problems with how it's done currently. If you were to do that, you'd need to build this package separately and so on which is definitely overkill